src/components/section/FolderList/FolderList.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from 'react';
+import { getFoldersById } from '@/pages/api/api';

```

Comment:
<img width="894" alt="image" src="https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/assets/59413128/0555faa9-38e7-495f-89dd-de67628a3742">
타입 에러 나네요. 이 부분 아래와 같이 수정하셔야 할 것 같아요
```suggestion
import { getFoldersById } from '@/pages/api/hello';
```